torch/_tensor_docs.py

```diff
@@ -1898,6 +1898,22 @@ def callable(a, b) -> number
 returns the zero_point of the underlying quantizer().
 """)
 
+add_docstr_all('q_scales',
+               r"""
+q_scales() -> Tensor
+
+Given a Tensor quantized by linear (affine) per-channel quantization,
+returns a Tensor of scales of the underlying quantizer().
+""")
+
+add_docstr_all('q_zero_points',
+               r"""
+q_zero_points() -> Tensor
```

Comment:
Please change the names to be consistent (see comment on q_scales and q_zero_points)